monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -103,890 +95,67 @@ class ReportPageComponent extends AuthComponent {
       });
   };
 
-  updateMapFromServer = () => {
-    this.authFetch('/api/netmap')
-      .then(res => res.json())
-      .then(res => {
-        res.edges.forEach(edge => {
-          edge.color = {'color': edgeGroupToColor(edge.group)};
-        });
-        this.setState({graph: res});
-        this.props.onStatusChange();
-      });
+  componentDidMount() {
+    const ztReportRefreshInterval = setInterval(this.updateZeroTrustReportFromServer, 8000);
+    this.setState({ztReportRefreshInterval: ztReportRefreshInterval});
+    this.updateMonkeysRunning().then(res => this.getReportFromServer(res));
```

Comment:
Should update all reports not only ZT. Doesn't matter how long it takes, it's not stone age and users shouldn't refresh the page (so it takes long anyways?) to see updated info.